README.md

```diff
@@ -0,0 +1,63 @@
+[![Build Status](https://travis-ci.org/segunolalive/helloBooks.svg?branch=master)](https://travis-ci.org/segunolalive/helloBooks)
```

Comment:
Good detailing in your readme file. It can be better and more comprehensive. Take a look at [this](https://github.com/vynessa/dman/blob/develop/README.md). 

Is documenting your API using any documentation  generator one of the requirements for this project? If no, i'd advice you look up [slate](https://github.com/lord/slate) or swagger for this.